FightPub2/src/test/java/ControllerTest.java

```diff
@@ -51,11 +54,12 @@ public void border() {
         assertEquals(50, char1.getxCoord(), "Hahmo seinän sisällä");
     }
     
+   
     @Test
-    void  characterCollission(){
-        controller.getPlayer1().setxCoord(1500);
-        controller.getPlayer2().setxCoord(1500);
-        assertEquals(false, controller.checkCollission(), "Hahmot ovat toistensä päällä");
+    void characterCollission(){
+        controller.getCharacter(1).setxCoord(1480);
+        controller.getCharacter(2).setxCoord(1500);
+        assertEquals(true, controller.checkCollision(), "Hahmot ovat toistensa päällä");
```

Comment:
Onko hahmon leveys varmasti yli 20 olisi selkeämpää jos ovat päällekkäin

Reply:
Leveys ,korkeus jne integer muuttujien arvot vielä vähän auki chatacterin osalta